src/core/sys/windows/mswsock.d

```diff
@@ -175,16 +177,15 @@ static if (_WIN32_WINNT > 0x501) {
     #endif
     +/
 
-    extern(Windows)
-    {
-        /*  WinXP+, Srv2k3+
-         *  ms-help://MS.MSDNQTR.2003FEB.1033/winsock/winsock/disconnectex_2.htm
-         */
-        BOOL DisconnectEx(SOCKET, LPOVERLAPPED, DWORD, DWORD);
-
-        /*  WinXP+, Srv2k3+
-         *  ms-help://MS.MSDNQTR.2003FEB.1033/winsock/winsock/wsarecvmsg_2.htm
-         */
-        int WSARecvMsg(SOCKET, LPWSAMSG, LPDWORD, LPWSAOVERLAPPED, LPWSAOVERLAPPED_COMPLETION_ROUTINE);
-    }
+    /*  Win Vista+, Srv2k3+
+     *  ms-help://MS.MSDNQTR.2003FEB.1033/winsock/winsock/disconnectex_2.htm
+     */
+    extern(Windows) BOOL DisconnectEx(SOCKET, LPOVERLAPPED, DWORD, DWORD);
```

Comment:
While I did not find DisconnectEx in any SDK header, the german MSDN page says it's available since WinXP:
https://msdn.microsoft.com/de-de/library/windows/desktop/ms737757(v=vs.85).aspx
while the US page restricts it to Vista:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms737757(v=vs.85).aspx
Maybe stay less restrictive and let it fail at link time.

Reply:
> Maybe stay less restrictive and let it fail at link time

As far as I can see, in [`src/core/sys/windows/w32api.d`](https://github.com/dlang/druntime/blob/v2.077.0/src/core/sys/windows/w32api.d), we're always defining `_WIN32_WINNT` to be `0x501`, since we never compile with version `Windows7` or `Windows8`, or `WindowsXXX`...
I wonder if it wouldn't be more practical to do away with the use of `_WIN32_WINNT` in druntime and unconditionally define all symbols and let them fail at link-time as you suggest for this particular case.

Reply:
This perhaps goes in line with @CyberShadow's suggestion to remove `version (Platform):` from the top of the header files, so you could for example use the definition of `BITMAPINFOHEADER` to read .bmp images on other platforms, instead of having to copy-paste the definition in your own project. (Though, AFAIR, a pull request to remove `version (Windows):` was blocked because it increased the size of druntime on Linux since now module info had to be emitted for more modules.)